src/server/PostManager.js

```diff
@@ -1,6 +1,34 @@
+import db from '../server/database';
+import {KeyUtils, ENTITY} from '../server/key-utils';
 
 export default class PostManager {
```

Comment:
1. Well... although it's not just about item and user, I think it still belongs to User API.
2. about the issue, the "type" is needed because of these relationship. so it belongs to the relationship.